cartridges/bm_algolia/cartridge/static/default/css/algolia.css

```diff
@@ -80,3 +80,34 @@ td {
 tr {
     padding: 0;
 }
+.info-icon {
+    display: inline-block;
+    margin-left: 5px;
+    cursor: pointer;
+    background-color: #007bff;
```

Comment:
This blue make the icons maybe a bit too visible. I'd make it more pale or smaller. Wdyt?

Reply:
Yes, you are right, Now they are smaller :)

Reply:
The color still seems far too vivid for me. The goal is to have the button visible, but not too intrusive.

Reply:
Great!